lib/gtfs/checkpoint.ex

```diff
@@ -0,0 +1,3 @@
+defmodule Gtfs.Checkpoint do
```

Comment:
I know that they're in a field called `checkpoint_id` in the GTFS file, but to avoid confusion they should be called Timepoints internally & externally.